lua/gatekeeperc.lua

```diff
@@ -8,40 +8,18 @@ struct gatekeeper_if {
 	char		**pci_addrs;
 	uint8_t		num_ports;
 	char		*name;
-	/*
-	 * The fields below are for internal use.
-	 * Configuration files should not refer to them.
-	 */
-	uint8_t		*ports;
-	uint8_t         id;
 };
 
 struct net_config {
 	uint16_t		num_rx_queues;
 	uint16_t		num_tx_queues;
 	struct gatekeeper_if	front;
 	struct gatekeeper_if	back;
```

Comment:
Given that LuaJIT will always miscalculate the position of the field back, and this patch already adds methods to safely obtain the fields front and back, the most appropriate action here is to move both fields to the hidden section of the struct. In effect, these two lines here in the `lua/gatekeeperc.lua` are going away.